lib/main_micro_alp.pm

```diff
@@ -170,6 +170,7 @@ sub load_remote_controller_tests {
 sub load_common_tests {
     loadtest 'console/regproxy' if is_regproxy_required;
     loadtest 'microos/networking';
+    loadtest 'microos/networkmanager';
```

Comment:
I would move this to `load_network_tests`, which is executed only in ALP (for now).

Reply:
sure, will do